src/Tizen.Account.AccountManager/Tizen.Account.AccountManager/Account.cs

```diff
@@ -72,6 +77,11 @@ public int AccountId
         {
             get
             {
+                if (AccountErrorFactory.IsAccountFeatureSupported() == false)
+                {
+                    AccountErrorFactory.ThrowNotSupportedException(AccountError.NotSupported, "account feature not supported");
+                }
```

Comment:
Do not need it, because if not support account Account object can not created

Reply:
This review comment is incorporated and changes are done in new patch.